src/main/java/com/myorg/mvc/controllers/UserRestController.java

```diff
@@ -0,0 +1,76 @@
+package com.myorg.mvc.controllers;
+
+import com.myorg.mvc.model.User;
+import com.myorg.mvc.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
```

Comment:
Why does RestController work with registration form and index??  Rest api should work only with json bodies and resources. Please do not mix rest api and spring mvc